episode3/README.md

```diff
@@ -20,13 +20,17 @@ describe("Regression tests", () => {
 ```
 
 ## Refactoring the add function
-Now we know the code works, we could refactor the `functions` into a
-version with [lamda][lamdas], this is the "Modern way" to define
-functions. They have a slightly different way how the scope works, but
-we get to that later 😅
 
-In short, we will refactor this:
+What is [refactoring][refactoring]?
+The process of refactoring is to structure software by applying a series of refactorings without changing its observable behaviour.
+The last part is important, without changing its observable behaviour. Behaviour can be observed
+with tests and this way we can make sure it does not change while we are structuring our code.
 
+Now we know the code works, we could refactor the `functions` into a version with [lamda][lamdas],
```

Comment:
the link should be `[lamdas][lamda]`

Reply:
Technically, both are incorrect. It should be `lambdas`. Changing it.